src/components/molecules/VenuePartygoers/VenuePartygoers.tsx

```diff
@@ -16,8 +20,16 @@ const filterVenuePartygoers = (partygoers: User[], venue: Venue) => {
 export const VenuePartygoers = () => {
```

Comment:
I know this is departing from our current pattern here, but i'm worried that by having this in molecules people might assume that they can just use it, yet the selector's aren't necessarily guaranteed to have been connected higher up.

I think we're too 'free' with where we connect into redux, and that adds a lot more complexity when we need to think about where a change may impact.

For this component, since it's used only in the `NavBar` at this stage, which appears to already have selectors/etc for the `venu`, `parentVenue`, etc; I think it would be nicer if we make these props on this component, and pass them down to this. Keeping this as a 'dumb' component (eg. only takes its data via props)

Since `partygoers` doesn't seem to be in `NavBar`, let's keep the selector for it in this component for now. We can think about it more in future as to what a better (if any) pattern might be for it.

```suggestion
interface VenuePartygoersProps {
  venue: ?TODO:TYPE?,
  parentVenue: ?TODO:TYPE?,
}

export const VenuePartygoers = ({ venue, parentVenue }) => {
```

Reply:
I wonder if we can inherit from `React.FunctionComponent` and enforce the inability to connect to the store from that inherited class.

Reply:
It's probably more of a job for eslint/similar I would think. If/when it becomes a need to enforce in any strong way. But for now I think just being mindful of it and when to use it is good.

Unless you mean to constrain things such that you can only use a certain component if it exists within a certain parent that provides a dependency, in which case, React's context's would be my immediate thought (and i have a few musing thoughts on how to use this pattern in an effective way bubbling in my brain already)

Reply:
[NB, Future Improvement]